lib/arctic/vendor/api.rb

```diff
@@ -47,6 +47,11 @@ def create_product(shop_id, product)
         response
       end
 
+      def ready_for_update_products(shop_id)
+        response = request :get, "shops/#{shop_id}/products/update_scheduled"
+        response.body['products']
```

Comment:
Why wouldn't the Core API endpoint not just return a flat array of the products? 

I assume this is what the endpoint returns now? 

```
{
  products: [
    { id: 1, ... },
    ...
  ]
}
```

Why doesn't it just return:

```
[
  { id: 1, ... },
  ...
]
```

Reply:
Ok. I'll make it without products